src/Aimeos/Shop/Controller/JqadmController.php

```diff
@@ -257,6 +257,17 @@ protected function createAdmin() : \Aimeos\Admin\JQAdm\Iface
 		$context = app( 'aimeos.context' )->get( false, 'backend' );
 		$context->setI18n( app( 'aimeos.i18n' )->get( array( $lang, 'en' ) ) );
 		$context->setLocale( app( 'aimeos.locale' )->getBackend( $context, $site ) );
+		
+		$siteManager = \Aimeos\MShop::create( $context, 'locale/site');
+
+		$filter = $siteManager->filter();
+		$filter->add([ 'locale.site.code' => $site ]);
+    
+		$siteItem = $siteManager->search( $filter->slice( 0 ) );
+		$siteConfig = $siteItem->getConfig()->toArray();
+		$siteConfig = array_pop( $siteConfig );
```

Comment:
This seems to be wrong because only one setting will be applied to the config instead of all settings

Reply:
The array returned from $site-Item->getConfig()->toArray() is like this:

[1] => [
  'theme/default/--ai-bg' => '#000000',
  'theme/default/--ai-bg-alt' => '#ffffff',
]

so array_pop returns the full settings array